packages/dashboard/src/CdcDashboard.tsx

```diff
@@ -251,9 +160,13 @@ export default function CdcDashboard({ configUrl = '', config: configObj = undef
 
   const getVizKeys = (_config): string[] => Object.keys(config?.visualizations || {})
```

Comment:
Not sure where this function was added, but the config reference needs to be the local reference (_config) instead of the global one:

  const getVizKeys = (_config): string[] => Object.keys(_config?.visualizations || {})